app/src/main/java/mozilla/lockbox/view/ItemListFragment.kt

```diff
@@ -171,6 +173,9 @@ class ItemListFragment : CommonFragment(), ItemListView {
         get() = Setting.ItemListSort.values()
 
     override fun updateItems(itemList: List<ItemViewModel>) {
+        if (isRefreshing) {
```

Comment:
this is more side-effect-y behavior not being driven (and therefore not unit-testable) by the `Presenter`. I'd like to have the `ItemListView` expose the hooks to stop refreshing so that the `Presenter` can do the business logic part.